apps/dashr-wind-steaming/Notebook/wind-speed-generation.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
Why is the `Notebook` directory in this branch? Let's :hocho: any Python-related files from `dashr-web-streaming`.